js/filter_type.js

```diff
@@ -18,16 +21,19 @@ $(document).ready(function(){
                 var dist_options = [];
 
                 if (sortOptions.includes(xAxis) && sortOptions.includes(yAxis) && (xAxis !== yAxis)){
+                    var submitButtonTxt = "<div id='sort-form'></div></br><button type='button' id='submit-scroll' class='btn' onclick='sortPlots()'>";
+                    submitButtonTxt += "&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbspSUBMIT &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</button>";
```

Comment:
Try to avoid doing things like this -> `...&nbsp...`
Try to achieve this via CSS. It's more scalable.

Reply:
Done